data-service/src/main/java/com/amazonaws/blox/dataservice/storage/model/DaemonEnvironmentLatestVersionDDBRecord.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservice.storage.model;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBVersionAttribute;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@DynamoDBTable(tableName = "DaemonEnvironmentLatestVersion")
+@Data
+@Builder
+//Required for builder because an empty constructor exists
+@AllArgsConstructor
+public class DaemonEnvironmentLatestVersionDDBRecord {
```

Comment:
So this is for keeping track of the latest version of the environment since we need to keep all the versions around to be able to roll back to a specific version. If we don't have this we cannot retrieve the latest version. Do you have a better idea on how to model this?